app/admin/views.py

```diff
@@ -19,28 +22,19 @@
 @admin.route('/<agency_ein>')
 def main(agency_ein=None):
     if not current_user.is_anonymous:
+        if agency_ein is None:
+            agency_ein = current_user.default_agency
         if current_user.is_super:
             agency_form = SelectAgencyForm(agency_ein)
             agency_ein = agency_ein or agency_form.agencies.choices[0][0]
-            user_form = ActivateAgencyUserForm(agency_ein)
-            active_users = Users.query.filter_by(
-                is_agency_active=True,
-                agency_ein=agency_ein
-            ).order_by(
-                Users.last_name.desc()
-            ).all()
-            agency_is_active = Agencies.query.filter_by(ein=agency_ein).one().is_active
+            user_form = ActivateAgencyUserForm(current_user.default_agency)
+            active_users = get_agency_active_users(agency_ein)
```

Comment:
Use agency instead of ein above so that we only query once and can use the Agencies.active_users property.